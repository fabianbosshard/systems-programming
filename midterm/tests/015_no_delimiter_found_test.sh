exec "$1"  format=@a,@b,@c delim="," <<EOF
Apple;5;7
Banana;3;9
EOF
