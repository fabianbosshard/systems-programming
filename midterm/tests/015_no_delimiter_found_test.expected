Apple;5;7,,
Banana;3;9,,
